bin/process-manifests.sh

```diff
@@ -215,24 +215,30 @@ function process_manifests() {
 
 
 >&2 echo "On branch $GIT_BRANCH"
-manifests=$( find $GEARS_DIR $BOUTIQUES_DIR -iname "*.json" )
+if [ -z "$GIT_COMMIT_SENTINEL" ]; then
+    >&2 echo "Using all manifests"
+    manifests=$( find $GEARS_DIR $BOUTIQUES_DIR -iname "*.json" )
+    >&2 echo "$manifests"
+else
+    >&2 echo "Using updated manifests"
+    manifests=$( git diff --name-only $GIT_COMMIT_SENTINEL | grep -e "^$GEARS_DIR/..*$" -e "^$BOUTIQUES_DIR/..*$" || true)
```

Comment:
The echo comments explain the intended difference. Maybe I don't understand the question? One grabs all the manifests, and the other grabs only manifests changed since the commit id recorded in `$GIT_COMMIT_SENTINEL`